rust/integration-tests/src/sdk_tests/ans_processor_tests.rs

```diff
@@ -0,0 +1,155 @@
+use aptos_indexer_testing_framework::sdk_test_context::SdkTestContext;
+use sdk_processor::{
+    config::{
+        db_config::{DbConfig, PostgresConfig},
+        indexer_processor_config::IndexerProcessorConfig,
+        processor_config::{DefaultProcessorConfig, ProcessorConfig},
+    },
+    processors::ans_processor::AnsProcessorConfig,
+};
+use std::collections::HashSet;
+
+pub async fn setup_ans_processor_config(
+    test_context: &SdkTestContext,
+    staring_version: u64,
+    txn_count: usize,
+    db_url: &str,
+) -> (IndexerProcessorConfig, &'static str) {
+    let transaction_stream_config =
+        test_context.create_transaction_stream_config(staring_version, txn_count as u64); // since this will be always 1, we can remove from the arg list
+    let postgres_config = PostgresConfig {
+        connection_string: db_url.to_string(),
+        db_pool_size: 100,
+    };
+
+    let db_config = DbConfig::PostgresConfig(postgres_config);
+
+    let ans_processor_config = AnsProcessorConfig {
+        ans_v1_primary_names_table_handle:
+            "0x1d5f57aa505a2fa463b7a46341913b65757e3177c46a5e483a29d953627bee62".to_string(),
+        ans_v1_name_records_table_handle:
+            "0x21a0fd41330f3a0a38173c7c0e4ac59cd51505f0594f64d3d637c12425c3c155".to_string(),
+        ans_v2_contract_address:
+            "0x867ed1f6bf916171b1de3ee92849b8978b7d1b9e0a8cc982a3d19d535dfd9c0c".to_string(),
+        default: DefaultProcessorConfig {
+            per_table_chunk_sizes: AHashMap::new(),
+            channel_size: 100,
+            deprecated_tables: HashSet::new(),
+        },
+    };
+
+    let processor_config = ProcessorConfig::AnsProcessor(ans_processor_config);
+    let processor_name = processor_config.name();
+    (
+        IndexerProcessorConfig {
+            processor_config,
+            transaction_stream_config,
+            db_config,
+            backfill_config: None,
+        },
+        processor_name,
+    )
+}
+
+#[allow(clippy::needless_return)]
+#[cfg(test)]
+mod tests {
+    use crate::{
+        diff_test_helper::ans_processor::load_data,
+        sdk_tests::{
+            ans_processor_tests::setup_ans_processor_config, run_processor_test,
+            setup_test_environment, validate_json, DEFAULT_OUTPUT_FOLDER,
+        },
+    };
+    use aptos_indexer_test_transactions::{
+        // IMPORTED_MAINNET_TXNS_214433968_ANS_LOOKUP,
+        // IMPORTED_MAINNET_TXNS_2739163_ANS_CURRENT_ANS_LOOKUP,
+        // IMPORTED_MAINNET_TXNS_275584410_ANS_CURRENT_ANS_PRIMARY_NAME,
+        IMPORTED_MAINNET_TXNS_1056780409_ANS_CURRENT_ANS_PRIMARY_NAME_V2,
+        IMPORTED_MAINNET_TXNS_303690531_ANS_LOOKUP_V2,
+        IMPORTED_MAINNET_TXNS_438536688_ANS_CURRENT_ANS_LOOKUP_V2,
+    };
+    use aptos_indexer_testing_framework::{cli_parser::get_test_config, database::TestDatabase};
+    use sdk_processor::processors::ans_processor::AnsProcessor;
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
nit: can we add description what event we are testing so that we can easily identify which cases have covered. I really like how renee did here:https://github.com/aptos-labs/aptos-indexer-processors/blob/main/rust/integration-tests/src/sdk_tests/token_v2_processor_tests.rs#L76-L87 !

Reply:
good call out